### DISCUSSION TITLE
Sending other provider scopes

### DISCUSSION THREAD

**Discussion by eshaham** [TheRealFlyingCoder/remix-auth-socials#11] [Ideas]
Hi, thanks for creating this module 🙏 
I was trying to request the Google Sheets scope when creating a `GoogleStrategy`, but apparently `scope` is typed to only `'openid' | 'email' | 'profile'`.
I think it would be great if the provider could accept additional scopes.
Wdyt?

**Comment by TheRealFlyingCoder** [comment]
Holy smokes I haven't been around to check discussions, this is totally resolved now for all strategies :)
